x-pack/plugins/secops/public/components/timeline/timeline.test.tsx

```diff
@@ -144,7 +144,7 @@ describe('Timeline', () => {
 
         wrapper
           .find('[data-test-subj="textFilter"]')
-          .first()
+          .at(1)
```

Comment:
Note: `.at(1)` has something to do with weirdness around styled components and regular components with regards with the number of children on the DOM. So, you sometimes have to use `.first()` or you have to use a `.at(1)` for the second child.